README.md

```diff
@@ -22,12 +22,12 @@
 
 👤 **Richard Opiyo**
 
-- 
+-
 - GitHub: [@richardoppiyo](https://github.com/richardopiyo)
 - Twitter: [@blessed_ricky](https://twitter.com/blessedricky)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/richardoppiyo)
 
-
+[Presentation video of the website]https://www.loom.com/share/a0dc6fdbbe1b40c7a28c74b23b49134e
 
 ## To show your support, kindly
 
```

Comment:
- [ ] I would suggest that you add a live link to the README of the github pages link where the project has been deployed. It will help users see your project in action without looking at the code 😄